internal/storage/migrations/0005_wallet_access.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up
+
+ALTER TABLE users ADD COLUMN "wallet_accessed_at" timestamp DEFAULT NULL;
```

Comment:
As there's not much else being accessed, I would simply call it `seen_at`

Reply:
but its not when the user was last seen. they could have been seen a bunch of times, but if they didn't call a query that uses the wallet, it won't be updated